lib/notesclub_web/live/notebook_live/show.ex

```diff
@@ -16,8 +17,15 @@ defmodule NotesclubWeb.NotebookLive.Show do
     path = String.replace(uri, ~r/https?:\/\/[^\/]+/, "")
     url = Paths.path_to_url(path) |> URI.decode()
     notebook = Notebooks.get_by!(url: url, preload: [:user, :repo])
-
-    {:noreply, assign(socket, notebook: notebook, clap_count: notebook.clap_count)}
+    username = get_username(notebook.user)
```

Comment:
When twitter_username is nil, we do not want to add `@` to the message because it's a github username (not Twitter's).

Something like this (not tested):

```suggestion
    name_or_username = if notebook.user.twitter_username, do: "@#twitter_username}", else: notebook.user.name || notebook.user.username
```